content/en/docs/setup/install/operator/index.md

```diff
@@ -75,11 +73,197 @@ accessible to `istioctl` by using this command:
+{{< /text >}}
+
+Another common customization is to select different namespaces for features and components. The following is an example
+of installation namespace customization:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  defaultNamespace: istio-system
+  security:
+    namespace: istio-security
+    components:
+      citadel:
+        namespace: istio-citadel
+{{< /text >}}
+
+Applying this file will cause the default profile to be applied, with components being installed into the following
+namespaces:
+
+- Citadel installed into istio-citadel namespace
+- All other security components installed into istio-security namespace
+- All other Istio components installed into istio-system namespace
+
+## Customize K8s settings using the IstioControlPlane API
+
+The IstioControlPlane API allows each component's k8s settings to be customized in a consistent way.
+
+### Identify the feature or component settings
+
+Each component has a [KubernetesResourceSpec](https://github.com/istio/operator/blob/9f80ecaea0f17dfd8a33d86d72f72da8861e7417/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto#L411),
+which allows the following settings to be changed. Use this list to indentify the setting to customize:
+
+1. [Resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container)
+1. [Readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
+1. [Replica count](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+1. [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+1. [PodDisruptionBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#how-disruption-budgets-work)
+1. [Pod annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [Service annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [ImagePullPolicy](https://kubernetes.io/docs/concepts/containers/images/)
+1. [Priority class name](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass)
+1. [Node selector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector)
+1. [Affinity and anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
+
+All of these K8s settings use the K8s API definitions, so [K8s documentation](https://kubernetes.io/docs/concepts/) can be used for reference. 
+
+### Configure the feature or component settings
+After you indentify the name of the feature or component from the previous list, you can use the API to set the values using a configuration overlay file.
+
+For example, use this overlay file ([samples/pilot-k8s.yaml](https://github.com/istio/operator/blob/release-1.3/samples/pilot-k8s.yaml))
```

Comment:
I think there is a more idomatic way to include docs from the repo, but I don't recall what it is.  Linking to a branch is not the right way.

Reply:
https://istio.io/about/contribute/creating-and-editing-pages/#links-to-github-files

Reply:
or https://istio.io/about/contribute/creating-and-editing-pages/#github

Reply:
Done.

Reply:
thanks @frankbu.

Reply:
`Use the overlay file @istio/operator/samples/pilot-k8s.yaml@`